spynnaker/pyNN/protocols/munich_io_spinnaker_link_protocol.py

```diff
@@ -43,16 +43,16 @@
 PUSH_BOT_MOTOR_UART_SHIFT = 0 + _OFFSET_TO_I
 
 
-def munich_key(I, F, D):
-    return (I << _OFFSET_TO_I) | (F << _OFFSET_TO_F) | (D << _OFFSET_TO_D)
+def munich_key(Instr, F, D):
+    return (Instr << _OFFSET_TO_I) | (F << _OFFSET_TO_F) | (D << _OFFSET_TO_D)
```

Comment:
Did you mean to make these changes?

Reply:
flake8 has a new (I think) error for "ambiguous variable name", so it's not allowing the use of "I" here, for example.  That's why this has changed, though if preferable we could work out how to skip it instead.

Reply:
Ah OK, no problem, just wondered!